src/posthog-persistence.js

```diff
@@ -140,11 +143,11 @@ PostHogPersistence.prototype.upgrade = function (config) {
         }
     }
 
-    if (this.storage === _.localStorage) {
-        old_cookie = _.cookie.parse(this.name)
+    if (this.storage !== cookieStore) {
+        old_cookie = cookieStore.parse(this.name)
 
-        _.cookie.remove(this.name)
-        _.cookie.remove(this.name, true)
+        cookieStore.remove(this.name)
+        cookieStore.remove(this.name, true)
```

Comment:
nit: this feels kind of dirty if moving from memory -> localStorage for example, but I think all is good and better be safe with the cookies...